build.gradle.kts

```diff
@@ -17,13 +17,20 @@ java {
 
 repositories {
     mavenCentral()
+    gradlePluginPortal()
+    maven("https://repo.spring.io/milestone")
```

Comment:
What do we need these repositories for? poi is not available in central repo?

Reply:
Fixed.